kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,197 @@
+            - name: ENDPOINTS_INFO_SENSITIVE
+              value: 'false'
+            - name: S2S_NAMES_WHITELIST
+              value: 'em_api,em_gw,ccd_gw,ccd_data,sscs,divorce_document_upload,divorce_document_generator,probate_backend,jui_webapp,pui_webapp,em_npa_app'
+            - name: CASE_WORKER_ROLES
+              value: 'caseworker-probate,caseworker-cmc,caseworker-sscs,caseworker-divorce'
+            - name: ENABLE_IDAM_HEALTH_CHECK
+              value: 'false'
+            - name: ENABLE_S2S_HEALTH_CHECK
+              value: 'false'
+            - name: APPINSIGHTS_INSTRUMENTATIONKEY
+              value: '${APPINSIGHTS_INSTRUMENTATIONKEY}'
+          resources:
+            requests:
+              memory: '1024Mi'
+              cpu: '200m'
+            limits:
+              memory: '2048Mi'
+              cpu: '2500m'
+          ports:
+            - containerPort: 4550
+              name: http
+          imagePullPolicy: Always
+---
+#
+# Service
+#
+apiVersion: v1
+kind: Service
+metadata:
+  name: ${SERVICE_NAME}
+  namespace: ${NAMESPACE}
+  labels:
+    app: ${SERVICE_NAME}
+spec:
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 4550
+  selector:
+    app: ${SERVICE_NAME}
+---
+#
+# INGRESS
+#
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ${SERVICE_NAME}
+  namespace: ${NAMESPACE}
+  annotations:
+    kubernetes.io/ingress.class: traefik
+spec:
+  rules:
+    - host: ${SERVICE_FQDN}
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: ${SERVICE_NAME}
+              servicePort: 80
+---
+#
+# DATABASE
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: shared-database-initdb
+  namespace: ${NAMESPACE}
+data:
+  initdb.sql: |
+    CREATE USER emannotationapp WITH PASSWORD 'emannotationapp';
+    CREATE DATABASE emannotationapp WITH OWNER = emannotationapp ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shared-database
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: shared-database
+  template:
+    metadata:
+      labels:
+        app: shared-database
+    spec:
+      containers:
+        - image: postgres:10.5-alpine
+          name: shared-database
+          resources:
+            requests:
+              memory: '256Mi'
+              cpu: '200m'
+            limits:
+              memory: '1024Mi'
+              cpu: '1000m'
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 5432
+              name: postgres
+          volumeMounts:
+            - mountPath: /docker-entrypoint-initdb.d
+              name: initdb
+              readOnly: true
+      volumes:
+        - name: initdb
+          configMap:
+            name: shared-database-initdb
+---
+# TODO Which shared database is this? Alec added it
```

Comment:
its the database for em-annotation-app 
created here:
https://github.com/hmcts/rpa-em-annotation-api/pull/5/files#diff-94d15c95f467ca128570c4629136bc51R139

Doesn't need to be called shared as here its not actually being shared, but up to you if you want to both renaming.

Where it was copied from there was a few dbs i believe